frontend/src/component/pages/shared/UserShortLinksSection.tsx

```diff
@@ -38,6 +38,7 @@ export class UserShortLinksSection extends Component<IProps> {
           <Table
             headers={['Long Link', 'Alias']}
             rows={this.createTableRows()}
+            colClassNames={['cell-long', 'cell-alias']}
```

Comment:
To recommend passing in `columnWidths` instead of `colClassNames` because `colClassNames` is implementation detail of table component. The component using relays on those classes may break when we change the internal of the table.

Reply:
The `colClassNames` property was added to `Table` component so that columns can have their widths styled in via SCSS, but if we're going to pass the widths directly instead, then `colClassNames` should be deleted and instead replaced with `columnWidths`, which will add the widths directly to the columns.

Reply:
@Coteh Please remove `colClassNames` to enforce encapsulation. Thank you!